<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsultantMainWindow</class>
 <widget class="QWidget" name="ConsultantMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="acceptRequestButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Принять</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelledRequestButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>520</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Отклонить</string>
   </property>
  </widget>
  <widget class="QLabel" name="mainLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Запросы на консультацию</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listOfRequests">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>561</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
